woocommerce/woocommerce-ios
# Add Encrypted Logging Support

Adds encrypted logging support

**To Test:**
- Download the installable Build
- Navigate to Settings > WooCommerce, and tap on the Woo logo four times to bring up the Crash Debug Menu
- Tap "Crash Immediately"
- Re-launch the app, and find your crash in Sentry.
- It should have an associated log ID
- Validate that the log can be read

**Update release notes:**
